[2024-04-15T15:40:32.073+0000] {processor.py:154} INFO - Started process (PID=18876) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:40:32.090+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:40:32.095+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:40:32.095+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:40:32.390+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:40:32.373+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:40:32.393+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:40:32.670+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.621 seconds
[2024-04-15T15:41:03.521+0000] {processor.py:154} INFO - Started process (PID=18944) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:41:03.527+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:41:03.533+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:41:03.533+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:41:03.571+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:41:03.560+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:41:03.578+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:41:03.631+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.118 seconds
[2024-04-15T15:41:33.937+0000] {processor.py:154} INFO - Started process (PID=19010) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:41:33.939+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:41:33.941+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:41:33.941+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:41:33.957+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:41:33.955+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:41:33.959+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:41:33.988+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.057 seconds
[2024-04-15T15:42:04.331+0000] {processor.py:154} INFO - Started process (PID=19084) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:42:04.333+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:42:04.335+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:42:04.335+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:42:04.359+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:42:04.355+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:42:04.361+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:42:04.406+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.085 seconds
[2024-04-15T15:42:34.677+0000] {processor.py:154} INFO - Started process (PID=19151) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:42:34.679+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:42:34.681+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:42:34.680+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:42:34.700+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:42:34.694+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:42:34.702+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:42:34.734+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.066 seconds
[2024-04-15T15:43:05.130+0000] {processor.py:154} INFO - Started process (PID=19229) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:43:05.133+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:43:05.135+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:43:05.135+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:43:05.154+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:43:05.150+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:43:05.155+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:43:05.191+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.073 seconds
[2024-04-15T15:43:35.350+0000] {processor.py:154} INFO - Started process (PID=19298) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:43:35.352+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:43:35.354+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:43:35.354+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:43:35.372+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:43:35.370+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:43:35.373+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:43:35.417+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.074 seconds
[2024-04-15T15:44:06.216+0000] {processor.py:154} INFO - Started process (PID=19377) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:44:06.218+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:44:06.220+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:44:06.220+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:44:06.239+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:44:06.236+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:44:06.240+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:44:06.271+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.067 seconds
[2024-04-15T15:44:36.763+0000] {processor.py:154} INFO - Started process (PID=19445) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:44:36.765+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:44:36.767+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:44:36.767+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:44:36.784+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:44:36.781+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:44:36.786+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:44:36.821+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.067 seconds
[2024-04-15T15:45:06.999+0000] {processor.py:154} INFO - Started process (PID=19524) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:45:07.002+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:45:07.005+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:45:07.005+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:45:07.025+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:45:07.022+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:45:07.027+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:45:07.083+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.094 seconds
[2024-04-15T15:45:37.364+0000] {processor.py:154} INFO - Started process (PID=19593) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:45:37.367+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:45:37.369+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:45:37.369+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:45:37.386+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:45:37.383+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:45:37.387+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:45:37.424+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.067 seconds
[2024-04-15T15:46:07.801+0000] {processor.py:154} INFO - Started process (PID=19671) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:46:07.803+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:46:07.805+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:46:07.805+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:46:07.821+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:46:07.818+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:46:07.822+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:46:07.861+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.066 seconds
[2024-04-15T15:46:38.187+0000] {processor.py:154} INFO - Started process (PID=19740) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:46:38.190+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:46:38.192+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:46:38.192+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:46:38.209+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:46:38.206+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:46:38.210+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:46:38.255+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.074 seconds
[2024-04-15T15:47:08.520+0000] {processor.py:154} INFO - Started process (PID=19819) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:47:08.522+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:47:08.523+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:47:08.523+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:47:08.538+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:47:08.536+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:47:08.539+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:47:08.569+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.055 seconds
[2024-04-15T15:47:38.845+0000] {processor.py:154} INFO - Started process (PID=19896) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:47:38.849+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:47:38.852+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:47:38.852+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:47:38.869+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:47:38.866+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:47:38.870+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:47:38.905+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.066 seconds
[2024-04-15T15:48:09.189+0000] {processor.py:154} INFO - Started process (PID=19965) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:48:09.191+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:48:09.193+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:48:09.192+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:48:09.209+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:48:09.207+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:48:09.210+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:48:09.244+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.061 seconds
[2024-04-15T15:48:39.928+0000] {processor.py:154} INFO - Started process (PID=20044) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:48:39.930+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:48:39.931+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:48:39.931+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:48:39.948+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:48:39.944+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:48:39.951+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:48:39.985+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.063 seconds
[2024-04-15T15:49:10.148+0000] {processor.py:154} INFO - Started process (PID=20113) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:49:10.151+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:49:10.152+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:49:10.152+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:49:10.170+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:49:10.167+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:49:10.171+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:49:10.218+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.077 seconds
[2024-04-15T15:49:40.646+0000] {processor.py:154} INFO - Started process (PID=20191) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:49:40.648+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:49:40.650+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:49:40.649+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:49:40.667+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:49:40.663+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:49:40.669+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:49:40.699+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.061 seconds
[2024-04-15T15:50:11.455+0000] {processor.py:154} INFO - Started process (PID=20261) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:50:11.458+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:50:11.460+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:50:11.460+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:50:11.477+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:50:11.474+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:50:11.478+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:50:11.511+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.063 seconds
[2024-04-15T15:50:42.102+0000] {processor.py:154} INFO - Started process (PID=20339) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:50:42.105+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:50:42.107+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:50:42.106+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:50:42.127+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:50:42.124+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:50:42.128+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:50:42.163+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.074 seconds
[2024-04-15T15:51:13.013+0000] {processor.py:154} INFO - Started process (PID=20417) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:51:13.015+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:51:13.017+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:51:13.017+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:51:13.032+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:51:13.030+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:51:13.034+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:51:13.068+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.064 seconds
[2024-04-15T15:51:43.823+0000] {processor.py:154} INFO - Started process (PID=20488) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:51:43.826+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:51:43.828+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:51:43.828+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:51:43.848+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:51:43.845+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:51:43.849+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:51:43.884+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.072 seconds
[2024-04-15T15:52:14.085+0000] {processor.py:154} INFO - Started process (PID=20567) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:52:14.086+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:52:14.088+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:52:14.088+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:52:14.113+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:52:14.110+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:52:14.115+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:52:14.163+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.090 seconds
[2024-04-15T15:52:44.759+0000] {processor.py:154} INFO - Started process (PID=20636) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:52:44.762+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:52:44.764+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:52:44.764+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:52:44.781+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:52:44.778+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:52:44.783+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:52:44.820+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.071 seconds
[2024-04-15T15:53:14.961+0000] {processor.py:154} INFO - Started process (PID=20714) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:53:14.964+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:53:14.966+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:53:14.966+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:53:14.985+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:53:14.982+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:53:14.987+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:53:15.032+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.079 seconds
[2024-04-15T15:53:45.286+0000] {processor.py:154} INFO - Started process (PID=20784) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:53:45.288+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:53:45.290+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:53:45.289+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:53:45.304+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:53:45.302+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:53:45.306+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:53:45.342+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.064 seconds
[2024-04-15T15:54:16.065+0000] {processor.py:154} INFO - Started process (PID=20862) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:54:16.067+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:54:16.069+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:54:16.069+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:54:16.084+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:54:16.081+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:54:16.086+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:54:16.123+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.066 seconds
[2024-04-15T15:54:46.877+0000] {processor.py:154} INFO - Started process (PID=20929) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:54:46.879+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:54:46.881+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:54:46.881+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:54:46.898+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:54:46.895+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:54:46.899+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:54:46.934+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.064 seconds
[2024-04-15T15:55:17.304+0000] {processor.py:154} INFO - Started process (PID=21007) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:55:17.307+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:55:17.310+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:55:17.309+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:55:17.331+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:55:17.326+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:55:17.333+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:55:17.374+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.076 seconds
[2024-04-15T15:55:47.947+0000] {processor.py:154} INFO - Started process (PID=21086) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:55:47.949+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:55:47.951+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:55:47.951+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:55:47.970+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:55:47.967+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:55:47.974+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:55:48.018+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.078 seconds
[2024-04-15T15:56:18.141+0000] {processor.py:154} INFO - Started process (PID=21154) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:56:18.144+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:56:18.147+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:56:18.146+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:56:18.162+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:56:18.160+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:56:18.163+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:56:18.197+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.065 seconds
[2024-04-15T15:56:49.072+0000] {processor.py:154} INFO - Started process (PID=21233) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:56:49.074+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:56:49.076+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:56:49.076+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:56:49.090+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:56:49.088+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:56:49.091+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:56:49.126+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.060 seconds
[2024-04-15T15:57:19.227+0000] {processor.py:154} INFO - Started process (PID=21303) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:57:19.229+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:57:19.231+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:57:19.231+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:57:19.250+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:57:19.247+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:57:19.252+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:57:19.283+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.066 seconds
[2024-04-15T15:57:49.448+0000] {processor.py:154} INFO - Started process (PID=21382) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:57:49.450+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:57:49.451+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:57:49.451+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:57:49.467+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:57:49.464+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:57:49.468+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:57:49.506+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.067 seconds
[2024-04-15T15:58:20.304+0000] {processor.py:154} INFO - Started process (PID=21451) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:58:20.306+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:58:20.308+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:58:20.308+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:58:20.327+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:58:20.323+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:58:20.328+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:58:20.360+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.063 seconds
[2024-04-15T15:58:51.266+0000] {processor.py:154} INFO - Started process (PID=21529) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:58:51.269+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:58:51.271+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:58:51.271+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:58:51.290+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:58:51.287+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:58:51.293+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:58:51.332+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.074 seconds
[2024-04-15T15:59:21.623+0000] {processor.py:154} INFO - Started process (PID=21606) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:59:21.626+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:59:21.627+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:59:21.627+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:59:21.646+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:59:21.642+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:59:21.648+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:59:21.692+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.076 seconds
[2024-04-15T15:59:52.358+0000] {processor.py:154} INFO - Started process (PID=21673) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:59:52.360+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T15:59:52.362+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:59:52.362+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:59:52.379+0000] {logging_mixin.py:120} INFO - [2024-04-15T15:59:52.375+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T15:59:52.380+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T15:59:52.416+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.068 seconds
[2024-04-15T16:00:22.641+0000] {processor.py:154} INFO - Started process (PID=21751) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:00:22.643+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:00:22.646+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:00:22.646+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:00:22.665+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:00:22.662+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:00:22.668+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:00:22.711+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.079 seconds
[2024-04-15T16:00:53.022+0000] {processor.py:154} INFO - Started process (PID=21820) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:00:53.023+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:00:53.025+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:00:53.025+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:00:53.051+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:00:53.047+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:00:53.053+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:00:53.090+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.075 seconds
[2024-04-15T16:01:23.460+0000] {processor.py:154} INFO - Started process (PID=21898) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:01:23.462+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:01:23.463+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:01:23.463+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:01:23.485+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:01:23.481+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:01:23.487+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:01:23.519+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.068 seconds
[2024-04-15T16:01:53.882+0000] {processor.py:154} INFO - Started process (PID=21967) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:01:53.884+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:01:53.886+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:01:53.886+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:01:53.906+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:01:53.903+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:01:53.907+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:01:53.946+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.071 seconds
[2024-04-15T16:02:24.456+0000] {processor.py:154} INFO - Started process (PID=22044) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:02:24.458+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:02:24.461+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:02:24.460+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:02:24.481+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:02:24.477+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:02:24.482+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:02:24.530+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.081 seconds
[2024-04-15T16:02:55.433+0000] {processor.py:154} INFO - Started process (PID=22114) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:02:55.438+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:02:55.441+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:02:55.441+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:02:55.462+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:02:55.457+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:02:55.464+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:02:55.504+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.076 seconds
[2024-04-15T16:03:26.179+0000] {processor.py:154} INFO - Started process (PID=22193) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:03:26.182+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:03:26.186+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:03:26.185+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:03:26.216+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:03:26.213+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:03:26.219+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:03:26.273+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.107 seconds
[2024-04-15T16:03:56.547+0000] {processor.py:154} INFO - Started process (PID=22271) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:03:56.549+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:03:56.551+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:03:56.551+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:03:56.571+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:03:56.569+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:03:56.572+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:03:56.602+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.061 seconds
[2024-04-15T16:04:27.552+0000] {processor.py:154} INFO - Started process (PID=22339) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:04:27.554+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:04:27.557+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:04:27.556+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:04:27.577+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:04:27.573+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:04:27.578+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:04:27.610+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.063 seconds
[2024-04-15T16:04:58.123+0000] {processor.py:154} INFO - Started process (PID=22416) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:04:58.125+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:04:58.128+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:04:58.128+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:04:58.156+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:04:58.152+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:04:58.157+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:04:58.276+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.159 seconds
[2024-04-15T16:05:28.385+0000] {processor.py:154} INFO - Started process (PID=22488) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:05:28.388+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:05:28.389+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:05:28.389+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:05:28.406+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:05:28.403+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:05:28.407+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:05:28.441+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.062 seconds
[2024-04-15T16:05:58.682+0000] {processor.py:154} INFO - Started process (PID=22565) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:05:58.684+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:05:58.686+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:05:58.686+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:05:58.701+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:05:58.699+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:05:58.703+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:05:58.739+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.064 seconds
[2024-04-15T16:06:28.878+0000] {processor.py:154} INFO - Started process (PID=22634) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:06:28.880+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:06:28.881+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:06:28.881+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:06:28.899+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:06:28.897+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:06:28.901+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:06:28.939+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.068 seconds
[2024-04-15T16:06:59.285+0000] {processor.py:154} INFO - Started process (PID=22711) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:06:59.288+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:06:59.290+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:06:59.290+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:06:59.304+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:06:59.302+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:06:59.306+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:06:59.340+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.062 seconds
[2024-04-15T16:07:29.550+0000] {processor.py:154} INFO - Started process (PID=22779) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:07:29.552+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:07:29.554+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:07:29.554+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:07:29.575+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:07:29.572+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:07:29.576+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:07:29.609+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.070 seconds
[2024-04-15T16:07:59.966+0000] {processor.py:154} INFO - Started process (PID=22856) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:07:59.968+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:07:59.971+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:07:59.970+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:07:59.990+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:07:59.988+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:07:59.992+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:08:00.048+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.090 seconds
[2024-04-15T16:08:30.233+0000] {processor.py:154} INFO - Started process (PID=22924) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:08:30.235+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:08:30.238+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:08:30.237+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:08:30.266+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:08:30.257+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:08:30.268+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:08:30.337+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.114 seconds
[2024-04-15T16:09:00.440+0000] {processor.py:154} INFO - Started process (PID=23001) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:09:00.443+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:09:00.445+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:09:00.445+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:09:00.463+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:09:00.460+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:09:00.467+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:09:00.517+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.084 seconds
[2024-04-15T16:09:31.227+0000] {processor.py:154} INFO - Started process (PID=23070) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:09:31.229+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:09:31.231+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:09:31.231+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:09:31.248+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:09:31.245+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:09:31.249+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:09:31.293+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.072 seconds
[2024-04-15T16:10:01.445+0000] {processor.py:154} INFO - Started process (PID=23148) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:10:01.447+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:10:01.451+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:10:01.450+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:10:01.473+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:10:01.470+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:10:01.474+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:10:01.516+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.079 seconds
[2024-04-15T16:10:31.708+0000] {processor.py:154} INFO - Started process (PID=23225) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:10:31.711+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:10:31.714+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:10:31.713+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:10:31.740+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:10:31.736+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:10:31.741+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:10:31.776+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.077 seconds
[2024-04-15T16:11:02.047+0000] {processor.py:154} INFO - Started process (PID=23292) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:11:02.049+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:11:02.050+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:11:02.050+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:11:02.066+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:11:02.063+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:11:02.067+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:11:02.098+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.057 seconds
[2024-04-15T16:11:32.261+0000] {processor.py:154} INFO - Started process (PID=23370) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:11:32.263+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:11:32.264+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:11:32.264+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:11:32.281+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:11:32.278+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:11:32.282+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:11:32.315+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.060 seconds
[2024-04-15T16:12:02.919+0000] {processor.py:154} INFO - Started process (PID=23436) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:12:02.921+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:12:02.923+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:12:02.922+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:12:02.946+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:12:02.942+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:12:02.947+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:12:02.981+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.072 seconds
[2024-04-15T16:12:33.730+0000] {processor.py:154} INFO - Started process (PID=23515) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:12:33.733+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:12:33.736+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:12:33.735+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:12:33.765+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:12:33.761+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:12:33.768+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:12:33.812+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.095 seconds
[2024-04-15T16:13:03.913+0000] {processor.py:154} INFO - Started process (PID=23585) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:13:03.915+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:13:03.917+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:13:03.917+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:13:03.931+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:13:03.929+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:13:03.932+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:13:03.966+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.061 seconds
[2024-04-15T16:13:36.242+0000] {processor.py:154} INFO - Started process (PID=23662) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:13:36.244+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:13:36.246+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:13:36.246+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:13:36.268+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:13:36.264+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:13:36.269+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:13:36.312+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.078 seconds
[2024-04-15T16:20:17.366+0000] {processor.py:154} INFO - Started process (PID=23675) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:20:17.372+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-15T16:20:17.374+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:20:17.374+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:20:17.406+0000] {logging_mixin.py:120} INFO - [2024-04-15T16:20:17.399+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-15T16:20:17.411+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-15T16:20:17.481+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.120 seconds
